README.md

```diff
@@ -75,7 +75,7 @@ Policies can be specified using the [Group Policy templates on Windows](https://
 | **[`Permissions`](#permissions)** | Set permissions associated with camera, microphone, location, and notifications.
 | **[`PictureInPicture`](#pictureinpicture)** | Enable or disable Picture-in-Picture.
 | **[`PopupBlocking`](#popupblocking)** | Configure the default pop-up window policy as well as origins for which pop-up windows are allowed.
-| **[`Preferences`](#preferences)** | Set and lock preferences.
+| **[`Preferences`](#preferences)** | Set and lock preferences. (NOTE: in order to use this, settings in the **[`Preferences (Deprecated)`](#preferences-deprecated)** section must be cleared.
```

Comment:
Can you remove this one? They actually do mix and match everywhere except the GPO.

So I want to add a comment in the GPO itself.